appengine/search/google-checks.xml

```diff
@@ -0,0 +1,223 @@
+<?xml version="1.0"?>
```

Comment:
In the future, you can add these to the file in the java-repo-tools/ directory. Then it will be easier to push back upstream and to the other samples.